contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1123,58 +1147,19 @@ $ oc adm create-error-template > errors.html
 ----
 ....
 
-* If your command contains multiple lines and uses callout annotations, you must comment out the callout(s) in the codeblock, as shown in the following example:
-+
-....
-To scale based on the percent of CPU utilization, create a `HorizontalPodAutoscaler` object for an existing object:
+// Similar guidance as with 1006
 
-[source,terminal]
-----
-$ oc autoscale <object_type>/<name> \// <1>
-  --min <number> \// <2>
-  --max <number> \// <3>
-  --cpu-percent=<percent> <4>
-----
-<1> Specify the type and name of the object to autoscale.
-<2> Optional: Specify the minimum number of replicas when scaling down.
-<3> Specify the maximum number of replicas when scaling up.
-<4> Specify the target average CPU utilization over all the pods, represented as a percent of requested CPU.
-....
+[[command-syntax-replaceable-values]]
+==== Command syntax for replaceable values
+To mark up command syntax, use the code block and wrap any replaceable values in angle brackets (`<>`) with the required command parameter, using underscores (`_`) between words as necessary for legibility.
 
-* Separate a command and its related example output into individual code blocks.
```

Comment:
I'm not sure if I'm missing this somewhere else, but I think this is still important to call out (about separating commands from example output)

Probably could go with the single command in a single block guidance above

Reply:
Thank you for your feedback, @bergerhoffer. For this one, I made some changes - moved it up alongside the guidance for [Source tags for terminal commands and output](https://github.com/abhatt-rh/openshift-docs/blob/contribute-codeblocks/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output) and added a link to the section that you suggested.

Also, separated the [guidance for source metadata](https://github.com/openshift/openshift-docs/pull/64877/files#diff-719cf3b36e5cfdc45393ebde8ff09fd9a56c239b5671fe1cea5b7f364d647125R1044-R1045) and updated it to reflect some recent changes.